src/libgeodecomp/misc/math.h

```diff
@@ -3,11 +3,11 @@
 
 #include <cmath>
 
-#if defined(_MSC_VER)
-inline int round(double d)
-{
-    return d >= 0.0 ? static_cast<int>(d + 0.5) : static_cast<int>(d - 0.5);
-}
-#endif
+// #if defined(_MSC_VER)
```

Comment:
Well, it fails horribly on AppVeyor. It did compile for me without - not sure what's going on.